app/graphql/mutations/batch_import/create_batch_import.rb

```diff
@@ -4,24 +4,32 @@ class Mutations::BatchImport::CreateBatchImport < Mutations::BaseMutation
   argument :priority, Enums::BatchImportPriorityEnum, required: false
   argument :signed_blob_id, String, "Signed blob ID generated via `/api/v1/uploads`", required: true
 
-  field :batch_import, Types::BatchImportType, null: false
+  field :batch_import, Types::BatchImportType, null: true
```

Comment:
`:batch_import` can be null?

Reply:
I see, on error. Hmm.

Reply:
Yeah, if validation of the submitted CSV data fails (and then errors are reported).